modules/openapi-generator/src/main/resources/swift5/modelInlineEnumDeclaration.mustache

```diff
@@ -1,7 +1,35 @@
-    {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} enum {{enumName}}: {{^isContainer}}{{dataType}}{{/isContainer}}{{#isContainer}}String{{/isContainer}}, {{#useVapor}}Content, Hashable{{/useVapor}}{{^useVapor}}Codable{{/useVapor}}, CaseIterable {
+    {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} enum {{enumName}}: {{^isContainer}}{{dataType}}{{/isContainer}}{{#isContainer}}String{{/isContainer}}, {{#useVapor}}Content, Hashable{{/useVapor}}{{^useVapor}}Codable{{/useVapor}}, CaseIterable{{^generateFrozenEnums}}{{#isInteger}}, CaseIterableDefaultsLast{{/isInteger}}{{#isFloat}}, CaseIterableDefaultsLast{{/isFloat}}{{#isDouble}}, CaseIterableDefaultsLast{{/isDouble}}{{#isString}}, CaseIterableDefaultsLast{{/isString}}{{#isContainer}}, CaseIterableDefaultsLast{{/isContainer}}{{/generateFrozenEnums}} {
         {{#allowableValues}}
         {{#enumVars}}
         case {{{name}}} = {{{value}}}
         {{/enumVars}}
         {{/allowableValues}}
+        {{^generateFrozenEnums}}
+        // This case is a catch-all generated by OpenAPI such that the enum is "non-frozen".
+        // If new enum cases are added that are unknown to the spec/client, they are safely
+        // decoded to this case. The raw value of this case is a dummy value that attempts
+        // to avoids collisions with previously specified cases.
+        {{#isString}}
+        case unknownDefaultOpenApi = "unknown_default_open_api"
+        {{/isString}}
+        {{#isContainer}}
```

Comment:
@jarrodparkes should this logic of the `{{#isContainer}}` be added to the file `modelEnum.mustache`?

Reply:
You are right 👍